core/keyserver/keyserver.go

```diff
@@ -418,6 +421,9 @@ func (s *Server) ListEntryHistory(ctx context.Context, in *pb.ListEntryHistoryRe
 
 	// TODO(gbelvin): fetch all history from trillian at once.
 	// Get all GetUserResponse for all revisions in the range [start, start + in.PageSize].
+	if in.PageSize > s.revisionPageSize {
+		in.PageSize = s.revisionPageSize
```

Comment:
Add documentation to `ListEntryHistoryRequest.PageSize` to mention that it is also affected by the server's revisionPageSize setting.

Reply:
Done